### DISCUSSION TITLE
Bug: skopeo tries to pull latest tag anyway when mirror from nested repository

### DISCUSSION THREAD

**Discussion by stolbikov-da** [quay/quay#1424] [Q&A]
Seems the Quay 3.6.0+ has some issues with mirroring images from nested repositories without latest tag.
Skopeo tries to pull latest tag regardless of what is written in the tag pattern field.
So, seems it is impossible to have nested repo without latest tag as mirror source repo.

tried to mirror with this tag pattern "x86_64-latest"
STDERR:
time="2022-06-29T14:58:15Z" level=fatal msg="Error parsing image name \"docker://registry.gitlab.com/gitlab-org/gitlab-runner/gitlab-runner-helper:latest\": reading manifest latest in registry.gitlab.com/gitlab-org/gitlab-runner/gitlab-runner-helper: manifest unknown: manifest unknown

**Comment by dmesser** [comment]
This is a known limitation in the way Quay currently uses `skopeo`. We should change it now that `skopeo` has evolved which is tracked here: https://issues.redhat.com/browse/PROJQUAY-2179
